recipes/glue-qt/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "glue-qt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/glue-qt-{{ version }}.tar.gz
+  sha256: 26a7f10e8bb58426512921355a0b9078698170fcc66504b2174969e1d1073d42
+
+build:
+  noarch: python
+  entry_points:
+    - glue = glue_qt.main:main
+  script: {{ PYTHON }} -m pip install . -vv
```

I believe we also need this for the "semi-arch" package.

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  string: win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [win]
  string: unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [unix]
```